pyscf/adc/uadc.py

```diff
@@ -1408,18 +1422,22 @@ def get_imds_ea(adc, eris=None):
             M_ab_a -= 0.25*lib.einsum('mlef,mlbd,adef->ab',t2_1_a, t2_1_a, eris_vvvv, optimize=True)
             M_ab_a -= 0.5*lib.einsum('mldf,mled,aebf->ab',t2_1_a, t2_1_a, eris_vvvv, optimize=True)
             M_ab_a += lib.einsum('mlfd,mled,aebf->ab',t2_1_ab, t2_1_ab, eris_vvvv, optimize=True)
-            temp_a = t2_1_a.reshape(nocc_a*nocc_a,nvir_a*nvir_a)
-            eris_vvvv = eris_vvvv.reshape(nvir_a*nvir_a,nvir_a*nvir_a)
-            temp = np.dot(temp_a,eris_vvvv)
             del eris_vvvv
-            temp = temp.reshape(nocc_a,nocc_a,nvir_a,nvir_a)
-            M_ab_a -= 0.25*lib.einsum('mlaf,mlbf->ab',t2_1_a, temp, optimize=True)
 
+            temp_a = adc.t2_vvvv_imds[0].copy()
```

Comment:
No need to define temp_a. Just use class object directly in lines 1429 and 1430 without copy